src/js/tech/html5.js

```diff
@@ -848,6 +848,7 @@ Html5.Events = [
   'play',
   'pause',
   'ratechange',
+  'resize',
```

Comment:
Makes sense. I agree with @heff on the ambiguity, `playerresize` sounds good to me.